sdcm/nemesis.py

```diff
@@ -1405,13 +1405,12 @@ def shuffle_list_of_disruptions(self):
 
         self.log.debug(f"nemesis stack BEFORE SHUFFLE is {self.disruptions_list}")
         random.Random(nemesis_seed).shuffle(self.disruptions_list)
-        self.log.debug(f"nemesis stack AFTER SHUFFLE is {self.disruptions_list}")
+        self.log.info(f"List of Nemesis to execute: {self.disruptions_list}")
 
     def call_next_nemesis(self):
-        self.log.info(f'Selecting the next nemesis out of stack {self.disruptions_list}')
+        self.log.debug(f'Selecting the next nemesis out of stack {self.disruptions_list}')
```

Comment:
I'm not sure printing the whole list again and again on each nemesis, really help anything.
I think it can be removed

Reply:
thanks, done

Reply:
It did help in a scenario when @fgelcer claimed that there is an issue with Sisyphus that is not proceeding to the next nemesis (@fgelcer what was the outcome of it??).

If it's too big, maybe we can just print the human readable name instead of the callable attribute.

Reply:
@roydahan , the issues is still open (#4261) and i didn't have the opportunity to get to it yet

Reply:
and yes, it will be much more helpful to print the human readable name...

Reply:
I changed to readable form instead of long form.
Still during execution debug level is used - after shuffling nemesis we log them at info level - I think logging it once is enough.